IHP/Controller/Context.hs

```diff
@@ -143,21 +143,29 @@ instance HasField "frameworkConfig" ControllerContext FrameworkConfig where
 --
 -- This can be useful to customize the log formatter for all actions of an app:
 --
--- > import IHP.Log.Types
+-- > -- Web/FrontController.hs
```

Comment:
Keep the code example, or link to the guide?

Reply:
Let's keep it here 👍 This way it can be understood from the API reference as well